release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -667,6 +667,146 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-10-monitoring-stack-components-and-dependencies"]
+==== Monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.23.0
+* Grafana to 8.3.4
+* kube-state-metrics to v2.3.0
+* node-exporter to 1.3.1
+* prom-label-proxy to 0.4.0
+* Prometheus to 2.32.1
+* Prometheus adapter to 0.9.1
+* Prometheus operator to 0.53.1
+* Thanos to 0.23.1
+
+[id="ocp-4-10-monitoring-new-metrics-targets-page-in-web-console"]
+==== New page for metrics targets in web the console
+
+A new *Metrics Targets* page in the {product-title} web console shows targets for default {product-title} projects and for user-defined projects. You can use this page to view, search, and filter the endpoints that are currently targeted for scraping, which helps you to identify and troubleshoot problems.
+
+[id="ocp-4-10-monitoring-components-use-tls-authentication-for-metrics"]
+==== Monitoring components updated to use TLS authentication for metrics collection
+
+With this release, all monitoring components are now configured to use mutual TLS authentication, rather than Bearer Token static authentication for metrics collection. TLS authentication is more resilient to Kubernetes API outages and decreases the load on the Kubernetes API.
+
+[id="ocp-4-10-monitoring-updated-cmo-to-use-global-tls-profile"]
+==== Cluster Monitoring Operator updated to use the global TLS security profile
+
+With this release, the Cluster Monitoring Operator components now honor the global `tlsSecurityProfile` settings. 
+The following components and services now use the TLS security profile:
+
+* Alertmanager pods (ports 9092 and 9097).
+* `kube-state-metrics` pod (ports 8443 and 9443).
+* `openshift-state-metrics` pod (ports 8443 and 9443).
+* `node-exporter` pods (port 9100).
+* Grafana pod (port 3002).
+* `prometheus-adapter` pods (port 6443).
+* `prometheus-k8s` pods (ports 9092 and 10902).
+* Thanos query pods (ports 9092, 9093 and 9094).
+* Prometheus Operator (ports 8080 and 8443).
+* `telemeter-client` pod (port 8443)
+
+If you have enabled user-defined monitoring, the following pods now use the profile:
+
+* `prometheus-user-workload` pods (ports 9091 and 10902).
+* `prometheus-operator pod` (ports 8080 and 8443).
+
+[id="ocp-4-10-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+* *New*
+** Added a `namespace` label to all Thanos alerting rules.
+** Added the `openshift_io_alert_source=”platform”` label to all platform alerts. 
+
+
+* *Changed*
+** Renamed `AggregatedAPIDown` to `KubeAggregatedAPIDown`.
+** Renamed `AggregatedAPIErrors` to `KubeAggregatedAPIErrors`.
+** Removed the `HighlyAvailableWorkloadIncorrectlySpread` alert. This alert is no longer needed because the Cluster Monitoring Operator ensures hard anti-affinity for all monitoring components.
```

Comment:
`ensures hard anti-affinity for all monitoring components` 
this is improper, it is only for "Alertmanager/Prometheus adapter/Prometheus/Thanos Querier"
also for components enabled user-defined monitoring, "Prometheus/Thanos Ruler"
please see from `Added hard anti-affinity rules and pod disruption budgets for certain components` section

Reply:
@bburt-rh correct